### ISSUE TITLE
Multiple Doorbell events?

### ISSUE THREAD

**Issue by Veldkornet** [briis/unifiprotect#274]
I have the doorbell connected to a UDMP. Recently (at some point not too long ago it was working properly), when someone rings the doorbell, it seems to "ring multiple times".

FYI I'm using the new doorbell event, not the binary sensor. 

So basically, I have Home Assistant (or, NodeRED inside HomeAssistant) send a doorbell sound to my Sonos Speakers, and send a snapshot over Pushover when the doorbell is pressed. 

I also have the doorbell connected via HomeBridge (outside of HomeAssistant) and that works fine. 

So basically this is what happens:
- press doorbell
- UniFi Protect app has 1 notification
- HomeKit has 1 notification 
- HomeAssistant plays the doorbell 3x
- HomeAssistant sends my pushover snapshot 3x

Now since not the snapshot and de bell sound use this doorbell event as the source action, I'm assuming here that multiple events are being sent. 

I don't think that the Protect app on the UDMP or in HomeAssistant has changed recently that I recall, so that's also strange.

**Comment by Veldkornet** [comment]
FYI, I've switched back to just using the binary sensor in the mean time and that works as expected.

**Comment by briis** [comment]
Glad to hear that. There is been a lot of work done on this in both 0.10.x and the upcoming 0.11 releases. So that will hopefully stay stabil.
